com.yubico.YubiOath.json

```diff
@@ -0,0 +1,268 @@
+		}
+	    ]
+	},
+	{
+            "name": "pycparser",
+            "buildsystem": "simple",
+	    "build-commands": [
+		"pip3 install --prefix /app --no-deps ."
+	    ],
+	    "sources": [
+		{
+		    "type": "archive",
+		    "url": "https://github.com/eliben/pycparser/archive/release_v2.19.tar.gz",
+		    "sha256": "893ec3d0f26c0faa7d0b992979940353a1682f86b5cd1228740637055b112379"
+		}
+	    ]
+	},
+	{
+            "name": "cryptography",
+            "buildsystem": "simple",
+	    "build-commands": [
+		"python3 setup.py install --prefix=/app"
+	    ],
+	    "sources": [
+		{
+		    "type": "archive",
+		    "url": "https://github.com/pyca/cryptography/archive/2.6.1.tar.gz",
+		    "sha256": "e6b77dddc068dcbb13c193602d7a40dc0bb348ceb107b14be083e42afa24ab83"
+		}
+	    ]
+	},
+	{
+            "name": "fido2",
+            "buildsystem": "simple",
+	    "build-commands": [
+		"pip3 install --prefix /app --no-deps ."
+	    ],
+	    "sources": [
+		{
+		    "type": "archive",
+		    "url": "https://github.com/Yubico/python-fido2/releases/download/0.5.0/fido2-0.5.0.tar.gz",
+		    "sha256": "e356c4e2ff136a29ea7f0bf82e679c2251fb246c4d459c3c91f84b93af6888de"
+		}
+	    ]
+	},
+	{
+            "name": "pyopenssl",
+            "buildsystem": "simple",
+	    "build-commands": [
+		"pip3 install --prefix /app --no-deps ."
+	    ],
+	    "sources": [
+		{
+		    "type": "archive",
+		    "url": "https://github.com/pyca/pyopenssl/archive/19.0.0.tar.gz",
+		    "sha256": "c5689bb12c210e3f996ee0a49cee8867f2afbf7e9beb84f3522f9dc8fdcb83ae"
+		}
+	    ]
+	},
+	{
+            "name": "yubikey-manager",
+            "buildsystem": "simple",
+	    "build-commands": [
+		"pip3 install --prefix /app --no-deps ."
+	    ],
+	    "sources": [
+		{
+		    "type": "archive",
+		    "url": "https://github.com/Yubico/yubikey-manager/releases/download/yubikey-manager-0.7.1/yubikey-manager-0.7.1.tar.gz",
+		    "sha256": "177bbf953b8557b8de68bacbbdfc56764e93733f337b2327e322cfc4ed4f7d18"
+		}
+	    ]
+	},
+        {
+            "name": "yubioath-desktop",
+            "buildsystem": "qmake",
+	    "post-install": [
+		"sed -i 's/Icon=yubioath/Icon=com.yubico.YubiOath/' resources/yubioath-desktop.desktop",
+		"install -Dm644 resources/yubioath-desktop.desktop /app/share/applications/com.yubico.YubiOath.desktop",
+		"install -Dm644 resources/icons/yubioath.png /app/share/icons/hicolor/128x128/apps/com.yubico.YubiOath.png"
+	    ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://developers.yubico.com/yubioath-desktop/Releases/yubioath-desktop-4.3.6.tar.gz",
+		    "sha256": "451d47f23d08dbe3630dcaa14bf11bbd8e894c86e9dee6264af597b5a4550468"
+                },
+		{
+		    "type": "patch",
+		    "path": "yubioath-desktop-path.patch"
+		}
+            ]
+        },
+	{
+	    "name": "metadata",
+	    "buildsystem": "simple",
+	    "build-commands": [
```

Comment:
It doesn't need to be a separate module, you can include it in `yubioath-desktop` and install in `post-install`.